run_models.py

```diff
@@ -0,0 +1,247 @@
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+
+
+"""
+
+ run_models: 
+ 
+ This is a wrapper script that calls the ska thermal models.
+
+This program is the Python replacement of the Perl program run_models.pl
+
+LR calls this program but differentiates between two cases:
+
+   if you use the -T switch with LR, then LR calls run_models with the -out switch
+           - switch argument is the OFLS Directory. 
+           - run_models sets the LR_DIR to the OFLS directory (e.g. TEST_MAY0525)
+           - PREVENTS run_models from copying the resultant files into the web directory
+
+   if you DO NOT use the -T switch with LR, then LR calls run_models without the --out switch.
+          - run_models creates a PRODUCTION ofls directory and uses that 
+   
+
+There are a number of changes between this version and the perl version.
+
+1) No Linux/hostname check
+
+2) The Perl version uses $lr_dir for both the thermal model --oflsdir AND the -out command line switches.  This
+    prevents the user from running a model  on an existing OFLS directory but putting the results elsewhere:
+    which one might do for testing purposes.
+
+    This version has 2 separate command line switches:  --oflsdir and --out and that allows test runs
+    to use an OFLS directory for input without writing into that official OFLS directory. If --out is not
+    supplied, then the output  will be written into the ofls directory.  This is to maintain backward
+    compatibility,
+
+    This means that if you want to put the output files anywhere other than the ofls directory you MUST
+    specify where they should go, withthe path switch.  The Perl version used the $path variable to
+    determine whether or not to copy the plot files to the web for display.  Since we have disassociated path from oflsdir
+    we need another way to prevent copies to the web page.  For that, we use the --web switch.
+
+
+ This program should fail any particular script and continue on if there 
+ is an issue. 
+
+
+  Usage:
+ Specifying a Test Directory:
+ 
+   /data/acis/LoadReviews/script/run_models.py load_name -h $hostname  ofls_dir break_string -nlet_file $nlet_file 
```

Comment:
This line is nearly identical to one a couple down for Production, with slightly different nomenclature (break_string vs break_str) and missing the "$" on ofls_dir and break_string; but notably this doesn't include the "out" switch which was indicated as being part of the call for a test run.